.env

```diff
@@ -0,0 +1,8 @@
+SERVICE_KNOWLEDGE_USERNAME=service-knowledge
+SERVICE_KNOWLEDGE_PASSWORD=password
+
```

Comment:
this file should not be provided. just copy the content and put it into a suitable section of the readme (installation/configuration part) and tell the developers to create .env file by themselves (basically hidden files with .xyz, should not be in the git)